### DISCUSSION TITLE
How to start contributing to Open Source?

### DISCUSSION THREAD

**Discussion by TriBlog** [community/community#37944] [Other Feature Feedback, Questions, & Ideas]
I am a Python Developer & Freelance Technical Writer. Can anyone suggest some repos, so that I can start contributing?

**Comment by jfullstackdev** [comment]
this might be for Python

https://github.com/MunGell/awesome-for-beginners#python

**Reply by queenofcorgis** [reply]
Thanks @jdevstatic 😄

**Comment by melvincwng** [comment]
Hey @TriBlog, I have a beginner-friendly open source organization that you can join & contribute if you're interested. Link is stated below (you can refer to the instructions in the README.md):

https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project

Happy open source contributing 👋🏻 !
